lightning/src/util/persist.rs

```diff
@@ -761,6 +727,34 @@ where
 	}
 }
 
+impl<K: Deref, L: Deref, ES: Deref, SP: Deref> MonitorUpdatingPersister<K, L, ES, SP>
+where
+	ES::Target: EntropySource + Sized,
+	K::Target: KVStore,
+	L::Target: Logger,
+	SP::Target: SignerProvider + Sized
+{
+	// Cleans up monitor updates for given monitor in range `start..=end`.
```

Comment:
triple slash for rustdoc?

```suggestion
	/// Cleans up monitor updates for given monitor in range `start..=end`.
```